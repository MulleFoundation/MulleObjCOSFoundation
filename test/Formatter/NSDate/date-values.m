//
//  main.m
//  archiver-test
//
//  Created by Nat! on 19.04.16.
//  Copyright Â© 2016 Mulle kybernetiK. All rights reserved.
//

#ifdef __MULLE_OBJC__
# import <MulleObjCOSFoundation/MulleObjCOSFoundation.h>
#else
# import <Foundation/Foundation.h>
#endif

//#import "MulleStandaloneObjCFoundation.h"




int   main( int argc, const char * argv[])
{
   NSDate            *date;

   date = [[[NSDate alloc] initWithTimeIntervalSince1970:0] autorelease];
   printf( "1970: %s (%0.17g)\n", [[date description] UTF8String], [date timeIntervalSinceReferenceDate]);

   date = [[[NSDate alloc] initWithTimeIntervalSinceReferenceDate:0] autorelease];
   printf( "ReferenceDate: %s (%0.17g)\n", [[date description] UTF8String], [date timeIntervalSinceReferenceDate]);

   return( 0);
}
