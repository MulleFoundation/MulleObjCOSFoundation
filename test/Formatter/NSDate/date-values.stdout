1970: 1970-01-01T00:00:00Z (-978307200)
ReferenceDate: 2001-01-01T00:00:00Z (0)
